.github/workflows/pr.yml

```diff
@@ -99,7 +99,3 @@ jobs:
     - name: Run generate script
       run: npm run deploy:flutter
       working-directory: importer
-
-    # TODO: https://github.com/microsoft/fluentui-system-icons/issues/34
-    # - name: Build Flutter library
-    #   uses: ?
```

Comment:
We should run the `flutter build` command here to ensure the library builds without failure?

Reply:
Added the dry run script to make sure that package is built properly but not publish. Let me know what you think about that :)

Reply:
ok, but for dry run, does it need the accessToken and refreshToken? Could we remove them if possible?

Reply:
It seems accessToken and refreshToken are required for the dry-run . Just tested without that and it failed.